source/MaterialXTest/Render.cpp

```diff
@@ -1399,6 +1350,48 @@ TEST_CASE("Render validation of test suite", "[render]")
     }
     ioTimer.endTimer();
 
+    // Create validators and generators
+#if defined(MATERIALX_BUILD_RENDERGLSL)
+    AdditiveScopedTimer glslSetupTime(profileTimes.glslTimes.setupTime, "GLSL setup time");
+
+    mx::GlslValidatorPtr glslValidator = createGLSLValidator("sphere.obj", glslLog);
+    mx::ShaderGeneratorPtr glslShaderGenerator = mx::GlslShaderGenerator::create();
+
+    mx::ColorManagementSystemPtr glslColorManagementSystem = mx::DefaultColorManagementSystem::create(glslShaderGenerator->getLanguage());
```

Comment:
Ths CMS initialization isn't quite the same it should test
```
if (options.cmsFiles.size() && options.cmsFiles.count(file))
```
If there are none then no CMS definitions exist.

Reply:
I didn't quite understand the reason for the cmsFiles option, specifying which test files needs a color management system. Should not all testing of all files have a CMS available, and then depending on the use of "colorspace" attributes inside the test use the CMS system or not.